playground/pages/Home.vue

```diff
@@ -11,12 +11,69 @@ const { data } = await storyblokApi.get("cdn/stories/vue", {
 
 const story = ref(null);
 story.value = data.story;
-
+console.log(story.value);
```

Comment:
necessary to keep?

Reply:
This code is located on our playground for debugging purposes